os/kernel/Kconfig

```diff
@@ -1031,3 +1031,9 @@ config LOG_DUMP_NUMBUFS
 	default 2
 
 endif # LOG_DUMP
+
+config SECURITY_LEVEL
```

Comment:
You still don't reply my  comment.
We have the se module. Do we need to add this in kernel module??

Reply:
Sorry for the late reply.
At first, I added this in kerenl module because I thought this was about assert log on/off.
However, since the variable has been changed to security_level now, it would be right to move to SE module as you said.
Thank you for review.